junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -244,6 +244,11 @@ public static boolean isStatic(Member member) {
 		return Modifier.isStatic(member.getModifiers());
 	}
 
+	public static boolean returnsVoid(Method method) {
+		//checking the class itself yields problems with bridge methods
+		return method.getReturnType().toString().equals("void");
```

Comment:
Works like a charm. Opted for the `Void.TYPE` variant which I find slightly less ugly.